docs/GraphRepresentation.md

```diff
@@ -0,0 +1,321 @@
+represents the average number of coinstallations per coinstalled add-on.
+The normalized count for (A,B) can be thought of as the contribution
+to this average from add-on A.
+This normalization is most effective when comparing add-ons B
+with similar total installation counts,
+of which some may be coinstalled more widely than others.
+
+This treatment is implemented as [`RowCount`](../taar_lite/recommenders/treatments.py#L87).
+
+
+### Total relevance normalization
+
+This treatment accounts for the popularity of an add-on in terms of
+its overall total number of installations,
+under the assumption that the more widely used an add-on is,
+the less likely it is to be relevant.
+
+For each add-on B coinstalled with a given add-on A,
+the coinstallation count for (A,B) is divided by
+the total number of installations of B.
+
+Given the adjacency matrix $C$,
+the treatment divides each each entry $C_{ab}$
+by the column sum for column $b$.
+In terms of the graph representation,
+the weight on each edge (A,B) leaving A is divided by
+the sum of the weights on all edges entering B.
+
+Intuitively, the normalized count for (A,B) represents
+the proportion of B's total installs contributed by profiles
+that also have A installed.
+Thus, the highest-scoring add-ons B are those which are more likely
+to be coinstalled with A than with other add-ons.
+We would expect this normalization to do a better job
+than the [add-on count normalization](#add-on-count-normalization)
+at controlling for the heavy skew of the distribution
+of overall install counts.
+
+As discussed [above](#graph-representation), in the raw coinstallation matrix,
+row sums are equal to column sums for each add-on
+and represent the total number of installs.
+For a general add-on relational graph, the matrix may no longer be symmetric,
+but the intuition behind the normalization still applies.
+In this case, we can view the column sum for B
+as an __aggregate relevance score__ over the universe of add-ons,
+as it is the sum of relevance scores for B
+in relation to each other add-on A.
+The normalization converts raw relevance scores to
+the proportion of aggregate relevance derived from add-on A.
+
+
+This treatment is implemented as [`RowSum`](../taar_lite/recommenders/treatments.py#L64).
+
+
+### Proportional total normalization
+
+This treatment is a rescaled version of the [total relevance normalization](#total-relevance-normalization).
+Rather than total installs, popularity is quantified in terms of
+the proportion of coinstalls associated with each add-on.
+Here, the assumption is that more relevant recommendations tend to account for
+a higher proportion out of the given add-on's coinstallations than it does
+for other add-ons.
+
+Given add-on A, the coinstallation count for (A,B) for each add-on B
+is first divided by the sum of all A's coinstallation counts.
+The total relevance normalization is then applied to the resulting proportions.
```

Comment:
Not exactly, since RowCount uses the number of non-zero entries to normalize. This one divides by row sum and then divides by column sum, so it's more like applying RowSum along the rows and then along the column.

We should really rename the treatment implementations, as I think the Row* naming scheme is one of the sources of confusion (and why I gave them more descriptive names in the doc).